examples/source/1.components/amp-script/index.html

```diff
@@ -158,6 +164,37 @@
         fetchCurrentTime();
       </script>
     </div>
+    <!--
+        ## Multiple fetches
+
+        In a container whose size can change, your code can make DOM changes until 5 seconds after the final `fetch()` completes. This example makes multiple calls to a slow API. It displays the result from each call when it returns.
+    -->
+    <div>
+      <amp-script layout="container" script="multi-fetch-script" class="sample">
+        <button id="multi-fetch">How slow is our API?</button>
+      </amp-script>
+      <script id="multi-fetch-script" type="text/plain" target="amp-script">    
+        const randomTime = () =>  Math.floor(Math.random() * 10) + 5;
```

Comment:
Why make it random? How about making 10 requests where each requests takes 1s longer. This way we can visualize the cut off.

Reply:
it could not be random. I think the randomness part makes it more fun to play with. But also, to me, it implies more clearly how that this works - that your code can make mutations until 5 seconds after the last fetch returns... not, say, after you make your last fetch, which is kind of what is implied to me by a linearly increasing set of delays.

Of course, I could keep on making changes for 4 seconds after the last fetch returns, but that seems to me to much.